src/modules/date/index.ts

```diff
@@ -192,10 +192,14 @@ export class SimpleDateModule extends SimpleModuleBase {
   }): Date {
     const { from, to } = options;
 
-    const fromMs = toDate(from, this.faker.defaultRefDate).getTime();
-    const toMs = toDate(to, this.faker.defaultRefDate).getTime();
-    const dateOffset = this.faker.number.int(toMs - fromMs);
+    const fromMs = new Date(from).getTime();
+    const toMs = new Date(to).getTime();
+
+    if (Number.isNaN(from) || Number.isNaN(to)) {
+      throw new FakerError('Both from and to are required.');
+    }
 
+    const dateOffset = this.faker.number.int(toMs - fromMs);
     return new Date(fromMs + dateOffset);
```

Comment:
IMO we could merge these lines into one/or simplify it:
```suggestion
    return new Date(this.faker.number.int({ from: fromMs, to: toMs }));
```
```suggestion
    const dateValue = this.faker.number.int({ from: fromMs, to: toMs });
    return new Date(dateValue);
```